ITTN-066.tex

```diff
@@ -194,6 +194,31 @@ \subsubsection{Discussion}
 As of k8s 1.25, \code{true} is the default value.
 If k8s \code{userns} support reaches ``GA`` in a future release, it is probable that setting \code{hostUsers} to \code{true} will become forbidden.
 
+
+\subsection{Running as Root}\label{sec:nonroot}
+
+Processes of containers SHALL NOT run as root. 
```

Comment:
I has considered this but I think it will catch a fair number of existing deployments.  I believe the RSP needs to do operations as uid 0 to prepare user home directories.  The enforcement mechanism for this would be to require that `runAsNonRoot` is `true` in the `PodSecurityContext`: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#podsecuritycontext-v1-core

While this is an issue today, k8s userns support is in alpha, and hopefully prior to the start of the operations we will be able to enable it.